src/main/scala/com/modelfabric/sparql/api/PrefixMapping.scala

```diff
@@ -281,6 +281,25 @@ class PrefixMapping {
     }
   }
 
+  def isPrefixed(uri : String): Boolean = {
```

Comment:
This won't work, because ':' will be in any IRI that starts with say 'http://'. We need a RegExp, such as:
```scala
  private val AbbreviatedIriRegExp = "(^[\\w][\\w\\-]*[\\w]):([\\w][\\w\\-]*[\\w]$)".r
```

(borrowed from elsewhere, you know where :-))

Reply:
this logic works for the purpose of this method, to tell whether an URI string is prefixed by any prefix defined in this PrefixMapping (assuming 'http' will never be used as a prefix). if it's 'http://xxx', the method will correctly return false.